Tests/Get-FileHardLink.Tests.ps1

```diff
@@ -0,0 +1,148 @@
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+BeforeAll {
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+    $script:testRoot = $null
+    $script:testNum = 0
+
+    function GivenHardlink
+    {
+        param(
+            [Parameter(Mandatory, Position=0)]
+            [String[]] $LinkPath,
+
+            [Parameter(Mandatory)]
+            [String] $ThatTargets
+        )
+
+        if( -not (Test-Path -Path $testRoot -PathType Container) )

```

Comment:
Remove this block. `$testRoot` is created in the `Init`, which gets created in the`BeforeEach` for each test.